PKGBUILD

```diff
@@ -0,0 +1,118 @@
+pkgname=skybian
+_pkgname=skybian
+pkgdesc="Packaged modifications to the skybian image, including scripts and utilities - debian package"
+pkgver='1.0.0'
+_pkgver=${pkgver}
+pkgrel=6
+_pkgrel=${pkgrel}
+arch=( 'any' )
+_pkgarches=('armhf' 'arm64' 'amd64')
+_pkgpath="github.com/skycoin/${_pkgname}"
+url="https://${_pkgpath}"
+makedepends=('dpkg') # 'go' 'musl' 'kernel-headers-musl' 'aarch64-linux-musl' 'arm-linux-gnueabihf-musl')
+depends=()
+_debdeps=""
+source=(
+#original to skybian
+"skybian-static.tar.gz"
+#Below are scripts and utilities introduced by the maintainer
+"skybian-script.tar.gz"
+"skybian-util.tar.gz"
+"skycoin.gpg"
+)
+#tar -czvf skybian-static.tar.gz static
+#tar -czvf skybian-script.tar.gz script
+#tar -czvf skybian-util.tar.gz util
+sha256sums=('f372a652a01bf2dcbe7c7c8606cbeb9778441390698cc8c10d42262148c5fe4b'
+            'e96120c8219eb12e77094702b9fa45dd8499309c9162cee03af074e2e5222186'
+            '45828704c32db393d4d75f09c454c34ba1472893c31dd79c697bb25df2a2589d'
+            'f2f964bb79541e51d5373204f4030dce6948d2d7862e345b55004b59b93d30e4')
+
+build() {
+  for i in ${_pkgarches[@]}; do
+   _msg2 "_pkgarch=$i"
+   local _pkgarch=$i
+   echo ${_pkgarch}
+
+#   [[ $_pkgarch == "amd64" ]] && export GOARCH=amd64 && export CC=musl-gcc
+#   [[ $_pkgarch == "arm64" ]] && export GOARCH=arm64 && export CC=aarch64-linux-musl-gcc
+#   [[ $_pkgarch == "armhf" ]] && export GOARCH=arm && export GOARM=6 && export CC=arm-linux-gnueabihf-musl-gcc
+#   #_ldflags=('-linkmode external -extldflags "-static" -buildid=')
+#   #create the srvpk binaries
+#   cd ${srcdir}/util
+#   go build -trimpath --ldflags '-s -w -linkmode external -extldflags "-static" -buildid=' -o ${_pkgarch}.srvpk .
+
+  #create control file for the debian package
+  echo "Package: ${_pkgname}" > ${srcdir}/${_pkgarch}.control
+  echo "Version: ${_pkgver}-${_pkgrel}" >> ${srcdir}/${_pkgarch}.control
+  echo "Priority: optional" >> ${srcdir}/${_pkgarch}.control
+  echo "Section: web" >> ${srcdir}/${_pkgarch}.control
+  echo "Architecture: ${_pkgarch}" >> ${srcdir}/${_pkgarch}.control
+  echo "Depends: ${_debdeps}" >> ${srcdir}/${_pkgarch}.control
+  echo "Maintainer: Skycoin" >> ${srcdir}/${_pkgarch}.control
+  echo "Description: ${pkgdesc}" >> ${srcdir}/${_pkgarch}.control
+  cat ${srcdir}/${_pkgarch}.control
+done
+}
+
+
+package() {
+
+  for i in ${_pkgarches[@]}; do
+  _msg2 "_pkgarch=${i}"
+  local _pkgarch=${i}
+   echo ${_pkgarch}
+  _msg2 'creating dirs'
+  #set up to create a .deb package
+  _debpkgdir="${_pkgname}-${pkgver}-${_pkgrel}-${_pkgarch}"
+  _pkgdir="${pkgdir}/${_debpkgdir}"
+  #########################################################################
```

Comment:
Seems to have over a lot of comments.

Reply:
i cleaned up the IMGBUILD comments but i had yet to do it for this PKGBUILD. Thanks for pointing that out.

Reply:
fixed